recipe/meta.yaml

```diff
@@ -202,6 +202,18 @@ outputs:
         # ensure only one test is left (namely, our hint)
         - python -c "import sys; lines=open('testlog').readlines(); sys.exit(0 if any('======== 1 failed' in x for x in lines) else 1)"
 
+  - name: scipy-typed
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('scipy', exact=True) }}
+        - scipy-stubs ~={{ version }}.0
```

Comment:
Is the `~=` syntax universally supported? It's not present in the Conda or Conda-forge docs as far as I can tell, and it's always less clear than `>=x.y,<x+1` so I tend to not try using it.